ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml

```diff
@@ -224,3 +227,24 @@ spec:
           value: "$(tasks.get-supported-versions.results.indices)"
         - name: container_digest
           value: "$(tasks.publish-to-ocp-registry.results.container_digest)"
+
+    - name: set-github-status-published
```

Comment:
Well, this task should forward any results of bundle publishing (either successful or not), retrieved from IIB after the publishing call-  Jira says, that it's blocked by ISV-993, and it's done before it.

Reply:
I added a extra results to the publishing step and reuse it as a status argument. I am going to work on the publishing step once we have a Kerberos account.

Reply:
When you get to the publishing step, please make sure that the status result of the step are within the list of accepted states of the github status API: https://docs.github.com/en/rest/reference/repos#create-a-commit-status

Reply:
yy, I am aware of it.